tests/codebuild/update_tests.sh

```diff
@@ -57,8 +58,9 @@ function verify_update_integration_tests
 #    $3: Filename of the hap test to update
 function update_hap_test()
 {
+  local random_string=$RANDOM
```

Comment:
Where does this come from?

Reply:
I found this in in several unofficial docs such as - 
1. https://www.geeksforgeeks.org/random-shell-variable-in-linux-with-examples/
2. https://stackoverflow.com/questions/8988824/generating-random-number-between-1-and-10-in-bash-shell-script

Other alternatives included using tools such as `shuf` or `tr` or using the official random number generator `/dev/random` (or `/dev/urandom`). But I tested the `RANDOM` shell variable locally as well as in the pipeline and it worked fine. We don't need it to be perfectly sampled etc, so it should be ok. 
 
Do you see any issues with using this ?

Reply:
Wow I've never heard of this before. That's super cool. There is still a chance that we have collisions between runs, though? I'm not sure if there is enough entropy in this alone?